tests/release/e2e-test-default-with-deployment.go

```diff
@@ -0,0 +1,224 @@
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   string(publicKey),
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+				NonBlocking: []string{
+					"tasks", "attestation_task_bundle", "java", "dc-metro-map",
+					"test", "hermetic_build_task.build_task_not_hermetic",
+					"buildah_build_task.dockerfile_param_external_source",
+					"step_image_registries.disallowed_task_step_image"},
+			},
+		}
+
+		// _, err = framework.AsKubeAdmin.CommonController.CreateConfigMap(cm, devNamespace)
+		// Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleasePlan(sourceReleasePlanName, devNamespace, applicationNameDefault, managedNamespace, "")
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleaseStrategy("mvp-deploy-strategy", managedNamespace, "deploy-release", "quay.io/hacbs-release/pipeline-deploy-release:main", releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategy) //(releaseStrategyDefaultName, managedNamespace, releasePipelineNameDefault, releasePipelineBundleDefault, releaseStrategyPolicyDefault, releaseStrategyServiceAccountDefault, paramsReleaseStrategy)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.GitOpsController.CreateEnvironment(releaseEnvironment, managedNamespace)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.ReleaseController.CreateReleasePlanAdmission(targetReleasePlanAdmissionName, devNamespace, applicationNameDefault, managedNamespace, releaseEnvironment, "", "mvp-deploy-strategy") //releaseStrategyDefaultName)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.TektonController.CreateEnterpriseContractPolicy(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy) //(releaseStrategyPolicyDefault, managedNamespace, defaultEcPolicy)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.TektonController.CreatePVCInAccessMode(releasePvcName, managedNamespace, corev1.ReadWriteOnce)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.CommonController.CreateServiceAccount(releaseStrategyServiceAccountDefault, managedNamespace, managednamespaceSecret)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.CommonController.CreateRole(roleName, managedNamespace, roleRules)
+		Expect(err).NotTo(HaveOccurred())
+
+		_, err = framework.AsKubeAdmin.CommonController.CreateRoleBinding("role-relase-service-account-binding", managedNamespace, "ServiceAccount", roleName, "Role", "role-m6-service-account", "rbac.authorization.k8s.io")
```

Comment:
can you use `release` in the name instead of `relase`?

Reply:
removed the CRD creation